### PR TITLE
Invitation rejection

### PR DESCRIPTION
**Pull request by gmethvin**

### CODE REVIEW

File: shoebox/app/com/keepit/controllers/admin/AdminInvitationController.scala [hunk 1]
```diff
@@ -67,8 +68,29 @@ class AdminInvitationController @Inject() (
...
+  def rejectUser(id: Id[SocialUserInfo]) = AdminHtmlAction { implicit request =>
+    val result = db.readWrite { implicit session =>
+      val socialUser = socialUserRepo.get(id)
+      for (user <- socialUser.userId.map(userRepo.get)) yield {
+        val invite = invitationRepo.getByRecipient(id).getOrElse(invitationRepo.save(Invitation(
+          createdAt = user.createdAt,
+          senderUserId = None,
+          recipientSocialUserId = socialUser.id.get
+        )))
+        (userRepo.save(user.withState(UserStates.ACTIVE)),
```
Comment (eishay) [inline shoebox/app/com/keepit/controllers/admin/AdminInvitationController.scala]: is the user state "active" if we reject him?

Comment (gmethvin) [inline shoebox/app/com/keepit/controllers/admin/AdminInvitationController.scala]: This is wrong. Thanks. Fixed.

Comment (eishay) [inline shoebox/app/com/keepit/controllers/admin/AdminInvitationController.scala]: A good test would have caught that. Do we have tests for the invitations?

Comment (gmethvin) [inline shoebox/app/com/keepit/controllers/admin/AdminInvitationController.scala]: No. I should add some.

**Pull request merged**